jerry-core/ecma/builtin-objects/ecma-builtin-object-prototype.c

```diff
@@ -203,6 +207,64 @@ ecma_builtin_object_prototype_object_property_is_enumerable (ecma_object_t *obj_
   return ecma_make_boolean_value (is_enumerable);
 } /* ecma_builtin_object_prototype_object_property_is_enumerable */
 
+#if ENABLED (JERRY_ESNEXT) && ENABLED (JERRY_BUILTIN_ANNEXB)
+/**
+ * The Object.prototype object's '__defineGetter__' and '__defineSetter__' routine
+ *
+ * See also:
+ *          ECMA-262 v11, B.2.2.2
+ *          ECMA-262 v11, B.2.2.3
+ *
+ * @return ECMA_VALUE_ERROR - if the operation fails,
+ *         ECMA_VALUE_UNDEFINED - otherwise
+ */
+static ecma_value_t
+ecma_builtin_object_prototype_define_getter_setter (ecma_object_t *obj_p, /**< this argument */
+                                                    ecma_string_t *prop_name_p, /**< property name */
+                                                    ecma_value_t accessor, /**< getter/setter function */
+                                                    bool define_getter) /**< true - defineGetter method
+                                                                             false - defineSetter method */
+{
+  /* 2. */
+  if (!ecma_op_is_callable (accessor))
+  {
+    return ecma_raise_type_error (ECMA_ERR_MSG ("Getter is not callable."));
+  }
+
+  ecma_object_t *accessor_obj_p = ecma_get_object_from_value (accessor);
+  ecma_ref_object (accessor_obj_p);
```

Comment:
I don't think we need to add a reference here, the caller will keep the object alive. Then we don't need the free on line 256 either.